httpx/client.py

```diff
@@ -61,6 +66,7 @@ def __init__(
         app: typing.Callable = None,
         raise_app_exceptions: bool = True,
         backend: ConcurrencyBackend = None,
+        additional_middlewares: list = None,  # TODO: is this the best way to inject mws?
```

Comment:
Can we rename this param to just middlewares?

Reply:
I considered it but thought it might convey that you can override the built-in middlewares so I added the "additional". Happy to change it if we don't think that's the case.